### DISCUSSION TITLE
sockjs-node/info net::ERR_CONNECTION_REFUSED

### DISCUSSION THREAD

**Discussion by hcxmj** [webpack/webpack#16206] [Q&A]
node: v16.16.0;
webpack: ^5.24.4;
webpack-cli: ^4.5.0;

`  devServer: {
    contentBase: path.join(__dirname, 'public'),
    historyApiFallback: true,
    open: true,
    hot: true,
    disableHostCheck: true,
    host: '0.0.0.0',
    port: '4000',
    watchContentBase: false,
    allowedHosts: 'all',
  },`

The deployment method is not the traditional static file index.html deployment, but generates docker to start the local service on the cloud server, and maps the domain name to the virtual machine to access the application. The following error occurs when the domain name is accessed

<img width="925" alt="image" src="https://user-images.githubusercontent.com/58494807/187424892-45e8300f-6dd3-444b-a26f-a33823d0bfe7.png">

**Comment by alexander-akait** [comment]
Maybe you have firewall,  check it
